dist/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="https://cdn.jsdelivr.net/npm/places.js@1.17.1"></script>
+  <script src="bundle.js" async></script>
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700&display=swap" rel="stylesheet">
+  <title>Weather App</title>
+</head>
+
+<body>
+  <div class="container">
+    <div id="search-bar">
+
+    </div>
+    <div class="search-bar">
+      <input type="text" id='autocomplete' />
+      <button id="search">Check Weather</button>
+    </div>
+
+    <script>
+      (function() {
+        var placesAutocomplete = places({
+          appId: 'plH5ZDEU8PVD',
+          apiKey: '44054bfaac8e96b587b8cf11662093fa',
+          container: document.querySelector('#autocomplete'),
+          templates: {
+            value: function(suggestion) {
+              return suggestion.name;
+            }
+          }
+        }).configure({
+          type: 'city',
+          aroundLatLngViaIP: false,
+        });
+      })();
+    </script>
```

Comment:
Let's move this to the `src` folder to prevent missing logic with the `index.html` file.